docs/getting-started-guides/kubeadm.md

```diff
@@ -8,12 +8,24 @@ li>.highlighter-rouge {position:relative; top:3px;}
 ## Overview
 
 This quickstart shows you how to easily install a secure Kubernetes cluster on machines running Ubuntu 16.04 or CentOS 7.
-The installation uses a tool called `kubeadm` which is part of Kubernetes 1.4.
+The installation uses a tool called `kubeadm` which is alpha as of Kubernetes 1.4.
 
 This process works with local VMs, physical servers and/or cloud servers.
 It is simple enough that you can easily integrate its use into your own automation (Terraform, Chef, Puppet, etc).
 
-**The `kubeadm` tool is currently in alpha but please try it out and give us [feedback](/docs/getting-started-guides/kubeadm/#feedback)!**
+kubeadm is not a complete provisioning tool; it is designed as a part of a larger provisioning
```

Comment:
The problem I'm trying to solve is that users are genuinely confused about when they should use kubeadm, and when they should use systems like GKE or kops.

I don't mean to denigrate kubeadm, I think there's actually a clear distinction to be drawn:

* If you want to build-your-own, use kubeadm and integrate with your systems
* If you just want a working kubernetes cluster on GCE, and can live with the opinions, you should probably use GKE
* If you just want a working kubernetes cluster on AWS, and can live with the opinions, you should probably use kops
* If you can't live with either of these GKE/kops restrictions, you should use kubeadm.

I don't want to confuse people with details of the how at this point, so I merged our paragraphs to be:

```
kubeadm assumes you have a set of machines (virtual or real) that are up and running.  It is designed
to be part of a larger provisioning system - or just for easy manual provisioning.  kubeadm is a great
choice where you have your own infrastructure (e.g. bare metal), or where you have an existing
orchestration system (e.g. Puppet) that you have to integrate with.
```